internal/server/server.go

```diff
@@ -0,0 +1,18 @@
+// Package server provides a method that handles every requests incoming and outcoming the recipe application
+package server
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func handleHelloWorld(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "%s", "Hello Dima")
+}
+
+// Server method handles all requests
+func Server() {
```

Comment:
Always try to think of parameters. You should never have a function with no parameters other than the main() function. Minimally, it should return an error (that could be nil, meaning no error). In this case a listenTo string will also work nicely.

```
func Server(listenTo string) error {

}
```

Reply:
Rarely a function will not return any error, when you are dealing with very simply string and number operations - but even in such cases other parameters are present.

Reply:
Ok... thanks for the feedback. If I get correctly We are making the port dynamic by passing in the parameter listenTo right?